src/main/java/org/xstefank/whitelist/AddUserCommand.java

```diff
@@ -0,0 +1,32 @@
+package org.xstefank.whitelist;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.xstefank.model.PersistentList;
+
+public class AddUserCommand implements Command {
+
+    public static final String NAME = "AddUserCommand";
+
+    private String commandRegex;
+
+    public AddUserCommand(String commandRegex) {
+        this.commandRegex = commandRegex;
+    }
+
+    @Override
+    public void process(JsonNode payload, PersistentList adminList, PersistentList userList) {
+        String authorUsername = WhitelistProcessing.getAuthorUsernameFromIssuePayload(payload);
```

Comment:
I'm not a big fan of very long names, I can see it's taking payload as argument ;)